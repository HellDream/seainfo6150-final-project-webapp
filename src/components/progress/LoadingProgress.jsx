import React from "react";
import styles from "./LoadingProgress.module.css";
const LoadingProgress = (props) => {
    return <div className={styles.container}>
        <div className={styles.loader}></div>
    </div>
}

export default LoadingProgress;