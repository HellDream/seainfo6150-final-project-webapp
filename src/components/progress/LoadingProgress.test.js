import React from 'react';
import LoadingProgress from './LoadingProgress';
import { render } from '@testing-library/react'

describe('LoadingProgress tests', () => {
    it('renders correctly', () => {
        const { container } = render(<LoadingProgress />);
        expect(container).toMatchSnapshot();
    });
});
  