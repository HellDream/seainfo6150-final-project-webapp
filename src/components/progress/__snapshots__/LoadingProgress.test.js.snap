// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingProgress tests renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="loader"
    />
  </div>
</div>
`;
